app/importers/californica_mapper.rb

```diff
@@ -22,9 +22,11 @@ class CalifornicaMapper < Darlingtonia::HashMapper
     normalized_date: "Date.normalized",
     publisher: "Publisher.publisherName",
     repository: "Name.repository",
-    location: "Coverage.geographic"
+    location: "Coverage.geographic",
+    named_subject: "Name.subject"
   }.freeze
 
+
```

Comment:
The reason the Travis build failed is because you added this blank line at line 29 in this file.  If you delete this blank line, the Travis build should restart, and hopefully pass this time.  Let me know if you need help doing that.

The reason you can't have a blank line here is because we are using a static analysis tool called rubocop that has a lot of strict rules about code style.  Even though your code is perfectly valid, it doesn't allow that extra blank line.  Those rules are configurable, so your dev team can discuss it and decide if they want to loosen up some of the rules.

Reply:
Excellent! I fixed the blank line :) and you paired with me to get my local copy back on track.